repository_test.go

```diff
@@ -11,11 +11,14 @@ import (
 	"github.com/stretchr/testify/mock"
 )
 
+var now NowTimeFunc
```

Comment:
looks like unused?

Reply:
It is used in many tests.
But I will modify it to use only `Now`.